src/components/ClusterManager.js

```diff
@@ -138,6 +135,7 @@ export const NewClusterModal = withModalDrawer({ width: 650 })(class NewClusterM
     this.state = {
       profile: matchingProfile ? matchingProfile.name : 'custom',
       jupyterUserScriptUri: '',
+      leoImageName: 'Default',
```

Comment:
TOL: Since the image URL is the underlying data that we're picking, it might end up being cleaner to have that as the state (especially once we get to the next part of specifying custom URLs).